serious_stuff/pubspec.yaml

```diff
@@ -23,6 +23,11 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  firebase_core: ^0.4.0+9
+
+  # Dependencies for the Firebase products
```

Comment:
Should this comment be moved above  line 26, above the firebase core dependency?

Reply:
Oh, good point, will do that